src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -69,31 +71,36 @@
-                                          token-symbol]))
-        selected-account? (rf/sub [:wallet/current-viewing-account-address])
-        token-owners      (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
-        send-params       (if selected-account?
-                            {:token       token-data
-                             :stack-id    :screen/wallet.accounts
-                             :start-flow? true
-                             :owners      token-owners}
-                            {:token-symbol token-symbol
-                             :stack-id     :wallet-stack
-                             :start-flow?  true
-                             :owners       token-owners})
-        bridge-params     (if selected-account?
-                            {:token            token-data
-                             :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                             :stack-id         :screen/wallet.accounts
-                             :start-flow?      true
-                             :owners           token-owners}
-                            {:token-symbol     token-symbol
-                             :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                             :stack-id         :wallet-stack
-                             :start-flow?      true
-                             :owners           token-owners})]
+  (let [token-symbol           (:token token)
+        token-data             (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
+                                               token-symbol]))
+        fiat-unformatted-value (get-in token [:values :fiat-unformatted-value])
+        has-balance?           (money/greater-than fiat-unformatted-value (money/bignumber "0"))
+        selected-account?      (rf/sub [:wallet/current-viewing-account-address])
+        token-owners           (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
+        send-params            (if selected-account?
+                                 {:token       token-data
+                                  :stack-id    :screen/wallet.accounts
+                                  :disabled?   (not has-balance?)
+                                  :start-flow? true
+                                  :owners      token-owners}
+                                 {:token-symbol token-symbol
+                                  :stack-id     :wallet-stack
+                                  :start-flow?  true
+                                  :owners       token-owners})
+        bridge-params          (if selected-account?
+                                 {:token            token-data
+                                  :bridge-disabled? (or :disabled?
+                                                        (not has-balance?)
+                                                        (send-utils/bridge-disabled? token-symbol))
```

Comment:
will always evaluate to `:disabled?` and the later checks will not run, since the keyword is truthy

Reply:
thanks for spotting!